semgrep/tests/e2e/test_semgrep_rules_repo.py

```diff
@@ -19,12 +30,8 @@ def test_semgrep_rules_repo(run_semgrep_in_tmp):
             "--test-ignore-todo",
             "semgrep-rules",
         ],
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        encoding="utf-8",
     )
 
-    if output.returncode != 0:
-        pytest.fail("Regression in Semgrep-Rules Repo" + output.stdout + output.stderr)
-
-    subprocess.check_output(["python3", "-m", "semgrep", "--validate", "semgrep-rules"])
+    _fail_subprocess_on_error(
```

Comment:
Particularly this one. Without the `--config` argument this gives:

```
$ python -m semgrep --validate  ~/r2c/semgrep-rules/
No config given and .semgrep.yml was not found. Try running with --help to debug or if you want to download a default config, try running with --config r2c
```

Since we were only doing `check_output` we weren't alerted to this issue.